includes/class-amp-story-post-type.php

```diff
@@ -2053,4 +2079,107 @@ private static function generate_export( $post_id ) {
 		unlink( $file );
 		die();
 	}
+
+	/**
+	 * Returns the definitions for the stories settings.
+	 *
+	 * @since 1.3
+	 *
+	 * @return array
+	 *
+	 * - meta_args array Arguments passed to `register_meta`; sanitize_callback is required.
+	 * - data      array Any additional data.
+	 */
+	public static function get_stories_settings_meta_definitions() {
+		return apply_filters(
+			'amp_stories_settings_meta_definitions',
```

Comment:
Is a filter here really needed? If so, it would need a docblock. But I think we could remove that filter.